### PR TITLE
add scraping for animal crossing fishes

### PR DESCRIPTION
**Pull request by dandaandaaaaaan**

### CODE REVIEW

File: src/sources/acnh/fish.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            sellPrice: 'td:nth-child(5)',
+            season: 'td:nth-child(7)',
+            time: 'td:nth-child(6)',
+          },
+        },
+      ],
+    },
+  ]);
+
+  for (var i = fishes.length -1; i >= 0 ; i--){
```
Comment (duncanleo) [inline src/sources/acnh/fish.js:35]: Could you use a `for of` loop instead?

File: src/sources/acnh/fish.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    if (fishes[i].name == null){
+        fishes.splice(i, 1);
+    }
+  }
+
+  const data = fishes.map(fish => Object.assign(fish, {type: 'Fish'}));
```
Comment (duncanleo) [inline src/sources/acnh/fish.js:41]: I think you can remove this

File: src/sources/acnh/model.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/**
+ * @typedef Item
```
Comment (duncanleo) [inline src/sources/acnh/model.js:2]: Nice!

File: src/sources/acnh/fish.js (hunk 2 above)
Comment (dandaandaaaaaan) [reply src/sources/acnh/fish.js:41]: is for adding of insects next time

File: src/sources/acnh/fish.js (hunk 1 above)
Comment (dandaandaaaaaan) [reply src/sources/acnh/fish.js:35]: ok

**Pull request merged**